localstack/services/s3/provider.py

```diff
@@ -74,6 +102,12 @@ def get_full_default_bucket_location(bucket_name):
     return f"{get_protocol()}://{bucket_name}.s3.{LOCALHOST_HOSTNAME}:{get_edge_port_http()}/"
 
 
+class InvalidArgumentError(CommonServiceException):
+    def __init__(self, message: str, name: str, value: str):
+        super().__init__("InvalidArgument", message, 400, False)
+        # TODO how to set values?
+
```

Comment:
@bentsku already added a batch for this exception [here](https://github.com/localstack/localstack/commit/34a71b4cbf3ba7c472ea55abfc560eb9a9889db0#diff-4b0f4fffc414ff51da92d57d0afdbedd29bfd8cb5683c57867fa2bf31e8753efR640). I added a few TODOs where we need to replace this exception, once the realted PR is merged.